server/country/countryRepository.js

```diff
@@ -117,6 +117,15 @@ const getRegions = async () => {
   return camelize(result.rows)
 }
 
+const getRegionGroups = async () => {
+  // Exclude Atlantis from region groups
+  const query = `
+        SELECT region_group, ARRAY_AGG(region_code) as regions FROM region  WHERE region_code != 'AT' GROUP BY region_group
```

Comment:
wouldn't it be better to fetch from region_group table so that you get the 'order' as well?
regions have the regionGroupId for the association in the UI